webapp/packages/core-app/src/Body.tsx

```diff
@@ -39,6 +39,21 @@ export const Body = observer(function Body() {
     document.documentElement.dataset.backendVersion = backendVersion;
   });
 
+  useEffect(() => {
+    const appLoadingScreen = document.getElementById('loading-screen');
+    let onTransitionEnd: () => void;
+
+    if (appLoadingScreen) {
```

Comment:
just check `if(!appLoadingScreen)`  after selector for preventing function execution